lib/gettext.ex

```diff
@@ -524,6 +524,9 @@ defmodule Gettext do
       messages are appended to the file. If `:sort_by_msgid` is set to `true`,
       existing and new messages will be mixed and sorted alphabetically by msgid.
 
+    * `:mark_as_obsolete` - a boolean that indicates if obsolete messages should
+      be deleted or marked as obsolete. By default messages are deleted.
+
```

Comment:
I think it might be clearer to have this option as `:on_obsolete`:

```suggestion
    * `:on_obsolete` - controls what happens when obsolete messages are found.
      If `:mark_as_obsolete`, messages are kept and marked as obsolete.
      If `:delete`, obsolete messages are deleted. Defaults to `:delete`.

```

Reply:
@whatyouhide Agreed, I will adapt it accordingly next week.